source/fuzz/transformation_add_bit_instruction_synonym.cpp

```diff
@@ -56,6 +56,11 @@ bool TransformationAddBitInstructionSynonym::IsApplicable(
     return false;
   }
 
+  // Signedness of the operands must be the same.
+  if (IsInstructionOperandsHasSameSign(ir_context, instruction)) {
```

Comment:
I think we can also move the two `if` statements above into this function.

Reply:
Done.